tests/system/test_system.py

```diff
@@ -68,8 +68,12 @@ def cleanup():
         operation()
 
 
-def test_collections(client):
+def test_collections():
+    from google.cloud import firestore
+
+    client = firestore.Client()
     collections = list(client.collections())
+
```

Comment:
We probably can just add an import into the existing test instead of writing the similar one

Reply:
My concern here is it looks redundant and may get removed in the future. If I saw this I would wonder why I couldn't use the global firestore reference.